code/modules/reagents/newchem/paradise_pop.dm

```diff
@@ -57,10 +124,36 @@
 	reagent_state = LIQUID
 	color = "#9933ff"
 
+/datum/reagent/grape_granade/on_mob_life(var/mob/living/M as mob)
+	if(!M)
+		M = holder.my_atom
+	if(prob(1))
+		var/turf/simulated/T = get_turf(M)
+		goonchem_vortex(T, 1, 1, 2)
+		M.emote("burp")
+		to_chat(M, "<span class='notice'>You feel ready to burst! Oh wait, just a burp...</span>")
+	else if(prob(25))
+		M.emote("burp")
```

Comment:
They do and it works. "<name> burps." Is all it does, no sounds or anything, just a simple emote like gasp.